packages/config-cli/src/cli/screenshot/screenshot.ts

```diff
@@ -75,9 +75,12 @@ export class CommandScreenShot extends CommandLineAction {
 
     let BasemapsServer: FastifyInstance | undefined = undefined;
     if (config != null) {
-      const port = await getPort({ port: portNumbers(10000, 11000) });
+      const port = await getPort();
       host = `http://localhost:${port}`;
-      BasemapsServer = await startServer(host, port, config, logger);
+      // Force a default url base so WMTS requests know their relative url
+      process.env[Env.PublicUrlBase] = host;
```

Comment:
if this is set we need to respect it we should not be overwriting env vars.

Reply:
We can set this back after the script running in finally.

Reply:
Proab safer to just use it if it exists the same as basemaps-server cli

Reply:
Cool, updated.